### PR TITLE
MBC migration mailing list and status history fix

### PR DESCRIPTION
**Pull request by ssettipalli**
MBC migration fixes to handle:
1. MBC mailing list with null first name / last name
2. Populate activity status history

Mailing list loading is already implemented. This PR includes just a fix tot handle null/empty names

### CODE REVIEW

**Comment by ssettipalli** [comment]
please run tests

File: pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java [hunk 1]
```diff
@@ -168,8 +168,13 @@ void loadMailingListData(Handle handle, JsonElement data, String studyCode) {
             firstName = getStringValueFromElement(thisEl, "firstname");
             lastName = getStringValueFromElement(thisEl, "lastname");
             email = getStringValueFromElement(thisEl, "email");
-            dao.insertByStudyGuidIfNotStoredAlready(firstName, lastName, email,
-                    studyCode, null, dateCreatedMillis);
+            if (StringUtils.isBlank(firstName)) {
+                firstName = " ";
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java]: Can the name be empty string, or does it have to be a space?

**Review by yufengwng — DISMISSED** [review]

File: pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java (hunk 1 above)
Comment (ssettipalli) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java]: as long as it passes DB NOT NULL constraint, we are good. 
updated to empty string ""

**Comment by ssettipalli** [comment]
please run tests

**Review by yufengwng — APPROVED** [review]

**Pull request merged**